.github/actions/ginkgo/main-prs.yaml

```diff
@@ -1,6 +1,6 @@
 # This file contain which k8s-versions should be executed for each PR
 ---
 k8s-version:
-  - "1.28"
+  - "1.29"
```

Comment:
Double-checking - Is it intentional to replace `1.28` instead of simply adding `1.29` in addition to it, here? Why not remove the oldest version instead? :thinking:

Reply:
@qmonnet yes, because we always want to test the oldest version that we support and the newest one that we support the one in the middle is for 5.4 kernel. Have in mind that this is just for PR runs, not for the "scheduled" jobs.

Reply:
Yep I realised after commenting that we still support 1.26. OK sounds good then, thanks!